src/typings/resize-observer.d.ts

```diff
@@ -0,0 +1,242 @@
+/**
+ * The **ResizeObserver** interface reports changes to the dimensions of an
```

Comment:
Can't we install this as a dependency?

Reply:
it was to avoid one more dependency for a native method, but yes no problem :)

Reply:
@JSReds I meant just for the typings, not for the polyfill, we should use the native method indeed. Surely the native resizeobserver will have typings?